auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -213,7 +216,7 @@ export default class AffiliatedEntityTable extends Vue {
       case (this.isNumberedIncorporationApplication(item)):
         return 'Pending'
       case (this.isTemporaryBusinessRegistration(item.corpType.code)):
-        return 'Pending'
+        return item.nrNumber
```

Comment:
This is a new property returned from affiliations request, to preserve NR number once the affiliation has moved into a draft filing.